EMacco/abarian-backend
# #171034967 verify account

#### What does this PR do?
This PR allows a user to be able to generate verification link and activate his/her account using that link

#### Description of Task to be completed?
- logged in user can send a new verification token
- verified users cannot request token to verify account
- verify token to make sure that it belongs to appropriate user
- change user status to verified

#### How should this be manually tested?
- checkout to this branch
- make a post request to `/auth/verify_account?email=email_address`
- an email should be sent to the user including the verification token
- copy the token and make a post request to `/auth/activate_user?email=email_address&token=verification_token`
- the account should be verified

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#171034967](https://www.pivotaltracker.com/story/show/171034967)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
